Task_TheFlexGame/index.css

```diff
@@ -1 +1,61 @@
 /* Start coding below for FLEX GAME TASK*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.header {
+  background-color: red;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  padding-top: 10px;
+  height: 100%;
+  flex: 1 1 0;
```

Comment:
The 'width' property should be removed as it is already set to 460px in line 10.